react/features/settings/components/web/ModeratorTab.tsx

```diff
@@ -46,26 +51,40 @@ export type Props = AbstractDialogTabProps & WithTranslation & {
      * enabled.
      */
     startVideoMuted: boolean;
+}
 
-    /**
-     * Invoked to obtain translated strings.
-     */
-    t: Function;
+const styles = (theme: Theme) => {
+    return {
+        container: {
+            display: 'flex',
+            flexDirection: 'column' as const
```

Comment:
shouldn't this be `as FlexDirection`?

Reply:
It could, but it doesn't really help. Generally for any CSS attribute that requires a specific type we just do `as const`. This error only appears when using `withStyles` HOC